src/pages/AddMembersPage/AddMembersPage.tsx

```diff
@@ -119,84 +139,67 @@ const AddMembersContents = ({
           Magic Link
         </TabButton>
         <TabButton
-          tab={Tab.ETH}
+          tab={Tab.CSV}
           currentTab={currentTab}
           setCurrentTab={setCurrentTab}
         >
-          ETH Address
+          CSV Import
         </TabButton>
-        {isFeatureEnabled('csv_import') && (
-          <TabButton
-            tab={Tab.CSV}
-            currentTab={currentTab}
-            setCurrentTab={setCurrentTab}
-          >
-            CSV Import
-          </TabButton>
-        )}
       </Flex>
       <Panel>
-        {currentTab === Tab.ETH && (
-          <Box>
-            <Text css={{ pb: '$lg', pt: '$sm' }} size={'large'}>
-              Add new members by wallet address.
-            </Text>
-
-            <NewMemberList
-              circleId={circle.id}
-              welcomeLink={welcomeLink}
-              revokeWelcome={revokeWelcome}
-            />
-          </Box>
-        )}
-        {currentTab === Tab.CSV && (
-          <Box>
-            <Box>CSV Import</Box>
-          </Box>
-        )}
-        {currentTab === Tab.LINK && (
-          <Box>
-            <Text css={{ pb: '$lg', pt: '$sm' }} size={'large'}>
-              Add new members by sharing a magic link.
-            </Text>
-            <ConstrainedBox css={{ mb: '$md' }}>
-              <Flex css={{ alignItems: 'center', mb: '$xs' }}>
-                <Text variant="label">
-                  Magic Circle Link
-                  {/*<ApeInfoTooltip>TODO: FILL THIS OUT</ApeInfoTooltip>*/}
+        <Box
+          css={{
+            width: '70%',
+            '@md': {
+              width: '100%',
+            },
+          }}
+        >
+          {currentTab === Tab.ETH && (
+            <Box>
+              <Text css={{ pb: '$lg', pt: '$sm' }} size={'large'}>
+                Add new members by wallet address.
+              </Text>
+              <NewMemberList
+                circleId={circle.id}
+                welcomeLink={welcomeLink}
+                revokeWelcome={revokeWelcome}
+                preloadedMembers={preloadedMembers}
+              />
+            </Box>
+          )}
+          {currentTab === Tab.LINK && (
+            <Box>
+              <Text css={{ pb: '$lg', pt: '$sm' }} size={'large'}>
+                Add new members by sharing a magic link.
+              </Text>
+              <MagicLink magicLink={magicLink} />
+            </Box>
+          )}
+          {currentTab === Tab.CSV && (
+            <Box>
+              <Text css={{ pb: '$lg', pt: '$sm' }} size={'large'}>
+                Please import a .CSV file with only these two columns: Name,
+                Address. &nbsp;
+                <Text size={'small'}>
```

Comment:
this should be the same size as the rest of the text

Reply:
ok i changed it back